api_app/connectors_manager/migrations/0003_auto_20210617_1035.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.4 on 2021-06-17 10:35
```

Comment:
since u are in developing mode of this, no need of multiple migration files; squash all migrations of the connectors_manager together into one.

Reply:
@sp35

Reply:
A easy to achieve this is to just delete all migrations and run makemigrations again.